package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "codeceptjs-browserstack",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "single": "npx codeceptjs run -c src/test/single/codecept.conf.js --steps",
+    "parallel": "npx codeceptjs run-multiple bstack -c src/test/parallel/codecept.conf.js --steps",
+    "local": "npx codeceptjs run -c src/test/local/codecept.conf.js --steps"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/princebaretto99/codeceptJS-browserstack.git"
```

Comment:
all urls should point to browserstack repo `https://github.com/browserstack/codecept-js-browserstack` rather than fork.